test/test-swaggerize-hapi.js

```diff
@@ -115,29 +115,28 @@ Test('test', function (t) {
 
     });
 
-    t.test('validation', function (t) {
-        t.test('query', function(t) {
-            var queryStringToStatusCode = {
-                'limit=2': 200,
-                'tags=some_tag&tags=some_other_tag': 200,
-                'limit=2&tags=some_tag&tags=some_other_tag': 200,
-                'limit=a_string': 400,
-                'unspecified_parameter=value': 400
-            }
+    t.test('query validation', function (t) {
+        var queryStringToStatusCode = {
+            'limit=2': 200,
+            'tags=some_tag&tags=some_other_tag': 200,
+            //'tags=billy,bob': 200,
```

Comment:
Yes - because I removed the `csv` style query params since hapi doesn't support them that way. Instead, the api spec has changed to `multi`, which hapi support.